test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/LotsUnloadTest.java

```diff
@@ -24,9 +24,10 @@
 
 /*
  * @test
- * @bug 8278602
+ * @bug 8278602 8313395
  * @summary Lots of classes being unloaded while we try to dump a dynamic archive
- * @requires vm.cds
+ * @requires vm.cds & !(vm.compMode == "Xcomp" & vm.graal.enabled)
```

Comment:
Any reasons to don't use probelmlist for this test exclusion? 
The 'requires' us usually for permanent test filtering.

Reply:
Permanent test filtering makes sense in this case since "the purpose of this test is neither to test libgraal nor Xcomp".

Reply:
I'll leave it up to you Ioi as to how to proceed. As the conversation in [JDK-8313395](https://bugs.openjdk.org/browse/JDK-8313395) indicates, this is not a libgraal problem.